source/_integrations/apprise.markdown

```diff
@@ -14,28 +14,48 @@ To use Apprise supported notifications, add the following to your `configuration
 ```yaml
 # Example configuration.yaml entry using URLs
 notify:
-  - platform: apprise
+  - name: NOTIFIER_NAME
+    platform: apprise
     url: YOUR_APPRISE_URLS
 ```
 
-You can also pre-define your own configuration files while storing them either remotely or locally. Simply just use the `config` option.
+Apprise URLs may seem strange at first, but they aren't complicated to create at all. They provide all of the information needed to activate a notification service in one single web-like URL. You specify more than one (causing them all to be notified) by just separating each URL with a comma and/or space.  To see how you can construct your own URL(s) [click here](https://github.com/caronc/apprise/wiki). For example, an [Apprise E-mail URL](https://github.com/caronc/apprise/wiki/Notify_email) configuration entry might look like this:
```

Comment:
We have now so much text above the `configuration` options table/list. That is not how it should be.

Please move the `configuration` table part right below the first example and put all other stuff below that.

Reply:
Done!